syntax = "proto3";

service Get_HyperParams{
    rpc get_hyperparams(param_request) returns (hyperparams) {}
}

message param_request{

}

message hyperparams{
    int64 N = 1;
    int64 M = 2;
    int64 t = 3;
    int64 T = 4;
}